test/unit/helper.spec.js

```diff
@@ -336,6 +340,12 @@ describe('Helpers Unit Tests', () => {
       const rate = helpers.calculateExchangeRate(100000, settings);
       expect(rate, 'to match', /6{1}[,.]9{1}0{1}/);
     });
+
+    it('should be infinite for unknown rate', () => {
+      settings.fiat = 'eur';
+      const rate = helpers.calculateExchangeRate(100000, settings);
+      expect(rate, 'to match', /∞/);
```

Comment:
Would this ever be the case, that the exchange rate is unknown?

Reply:
No, but the user's machine might not have an internet connect to fetch a current exchange rate. So we need to handle this case and display something.